content/chainguard/chainguard-images/about/differences-development-production.md

```diff
@@ -13,7 +13,7 @@ tags: ["Chainguard Images", "Product", ]
 images: []
 menu:
   docs:
-    parent: "chainguard-images"
+    parent: "about"
```

Comment:
it looks like this was not cleaned up with the move, not sure how important it is but just to clean it up